cypress/component/Select.spec.tsx

```diff
@@ -305,6 +305,16 @@ describe('Select', () => {
         );
       });
     });
+    context('when spacebar is typed and no value exists', () => {
+      beforeEach(() => {
+        cy.findByRole('combobox').focus();
+        cy.focused().realType(' ');
+      });
+      it('should open the menu', () => {
+        // should open the menu
+        cy.findByRole('listbox').should('exist');
+      });
+    });
```

Comment:
hmm, i think this creeped in when i merged master in my branch 😬

Reply:
not sure if prerelease/minor gets updated with master or if i should do a commit to remove this?